.github/workflows/ci.yml

```diff
@@ -192,6 +236,56 @@ jobs:
       - name: Test
         run: npm run test
 
+  workspaces-unit-tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest']
+        nodejs: ['18', '20', '21', '22', '23', '24']
+        include:
+          - os: 'macos-latest'
+            python_version: '3.11'
+          - os: 'windows-2022'
+            nodejs: '18.20.1'
+          - os: 'windows-2022'
+            nodejs: '20.12.1'
+          - os: 'windows-2022'
+            nodejs: '21.7.2'
+          - os: 'windows-2022'
+            nodejs: '22.10.0'
+          - os: 'windows-2022'
+            nodejs: '23.0.0'
+          - os: 'windows-2022'
+            nodejs: '24.0.0'
+    services:
+      neo4j:
+        image: ${{ (matrix.os == 'ubuntu-latest') && 'neo4j:4.4.42' || '' }}
+        ports:
+          - 11011:7687
+        env:
+          NEO4J_AUTH: neo4j/test_pw
```

Comment:
neo4j service is used for testing the neo4j instrumentation, I think setting up neo4j can be removed from this job?